log/async_file_writer.go

```diff
@@ -10,32 +10,43 @@ import (
 	"time"
 )
 
-type HourTicker struct {
+type TimeTicker struct {
 	stop chan struct{}
 	C    <-chan time.Time
 }
 
-func NewHourTicker() *HourTicker {
-	ht := &HourTicker{
+// NewTimeTicker notifies on a daily basis if dailyRotate is true, otherwise on an hourly basis
+func NewTimeTicker(dailyRotate bool) *TimeTicker {
+	ch := make(chan time.Time)
+	ht := TimeTicker{
```

Comment:
Since this could be a day ticker or hour ticker, the variable should rather be named `tt` ?